src/main/scala/uk/gov/nationalarchives/notifications/messages/EventMessages.scala

```diff
@@ -370,7 +370,7 @@ object EventMessages {
   private def generateSnsExportMessageBody(bucketName: String,
                                            consignmentRef: String,
                                            producer: Producer): SnsMessageDetails = {
-    val topicArn = eventConfig("sns.topic.transform_engine_v2_in")
+    val topicArn = eventConfig("sns.topic.da_event_bus_arn")
```

Comment:
You will need to update the `eventConfig` val in `Messages.scala` (line 36) to replace the `sns.topic.transform_engine_v2_in` entry with `da_event_bus_arn` entry. That should fix the unit tests as well, which are failing.

Reply:
OK thanks